services/executionengine/src/main/java/tech/pegasys/teku/services/executionengine/ExecutionEngineConfiguration.java

```diff
@@ -81,5 +93,11 @@ public Builder specProvider(final Spec spec) {
       this.spec = spec;
       return this;
     }
+
+    public Builder jwtSecretFile(final String jwtSecretFile) {
+      checkNotNull(jwtSecretFile);
+      this.jwtSecretFile = Optional.of(jwtSecretFile).filter(f -> !f.isBlank());
```

Comment:
This will need to support `jwtSecretFile` being `null` since that's the default value in the cli definition. This is what's causing most of the test failures. And we can simplify a little by using the isNotBlank method from commons-lang3. So probably:
```suggestion
      this.jwtSecretFile = Optional.ofNullable(jwtSecretFile).filter(StringUtils::isNotBlank);
```